driver/bpf/fillers.h

```diff
@@ -7111,4 +7111,90 @@ FILLER(sched_prog_fork_3, false)
 }
 #endif
 
+FILLER(sys_prctl_x, true)
+{
+	int val;
+	unsigned long option;
+	unsigned long arg;
+	unsigned long arg2;
+	int res;
+	long retval;
+
+	retval = bpf_syscall_get_retval(data->ctx);
+	res = bpf_val_to_ring(data, retval);
+	if (res != PPM_SUCCESS)
+		return res;
+
+	/*
+	 * option
+	 */
+	option = bpf_syscall_get_argument(data, 0);
+	res = bpf_val_to_ring(data, option);
+	if (res != PPM_SUCCESS)
+		return res;
+
+	/*
+	 * arg2
```

Comment:
To be honest I would keep just `arg2str` and `arg2int` and I would remove all `arg3`, `arg4`, `arg5`, ecc...
so in the end just 4 params:
* `res`
* `arg1`
* `arg2_str`
* `arg2_s64`

Reply:
I think that this is a good idea! These 3 params are not so useful! 😄